sections/shared/Layout/SideNav/SideNav.tsx

```diff
@@ -72,10 +71,7 @@ const SideNav: FC<SideNavProps> = ({ isDesktop }) => {
 			{menuLinks.map(({ i18nLabel, link, subMenu }, i) => {
 				const onSetSubMenuConfiguration = () => {
 					setSubMenuConfiguration({
-						// Debt data only exists on mainnet for now, need to hide otherwise
-						routes: (isMainnet
-							? subMenu
-							: subMenu!.filter(({ subLink }) => subLink !== ROUTES.Debt.Home)) as any,
+						routes: subMenu as any,
```

Comment:
So by removing this condition, you'd enable the debt page for testnets too. Which I am not against, but we just need to make sure the graph is deployed for Kovan.